xwiki-platform-core/xwiki-platform-url/xwiki-platform-url-api/src/main/java/org/xwiki/url/URLConfiguration.java

```diff
@@ -47,4 +51,15 @@ default boolean useResourceLastModificationDate()
     {
         return true;
     }
+
+    /**
+     * @return the list of trusted domains that can be used in the wiki.
```

Comment:
I think a definition of what is a trusted domain would be nice since otherwise the javadoc is not really interesting ;)